java/code/src/org/cobbler/CobblerObject.java

```diff
@@ -171,7 +171,7 @@ protected static List<Map<String, Object>> lookupDataMapsByCriteria(
         Map<String, String> criteria = new HashMap<>();
         criteria.put(critera, value);
         return (List<Map<String, Object>>)
-                client.invokeTokenMethod(findMethod, criteria);
+                client.invokeTokenMethod(findMethod, criteria, false);
```

Comment:
This seems to be wrong. Paramater needs to be "true": See also https://github.com/SUSE/spacewalk/pull/22954

Reply:
@mcalmer I see you have added few more commits to this PR after you did this review.

Is this PR ready to be merged now?

Thanks in advance!